spec/requests/post_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+RSpec.describe PostsController, type: :request do
+  describe 'Check post main page' do
+    before(:example) do
+      get '/users/1/posts'
+      expect(response).to render_template(:index)
```

Comment:
- [ ] The `expect` should be in its own **`it`** block. Kindly remove it from here.

Reply:
Thanks a lot @DammyShittu  for this observation, I'll do well to fix it immediately👍.